commons/src/lib.rs

```diff
@@ -104,6 +116,10 @@ pub fn validate_content_type(
     let full_type = header::HeaderValue::from_static(content_type);
```

Comment:
We should make up our mind about whether `validate_content_type` accepts supported types from the caller (which is how `content_type` is currently working) or via an internal global (which is how you're setting up the `CINCI_VERSION` consumption).  Personally I'd rather have callers pass this in explicitly, but to do that they'd need to build a vector with `application/json` and the set of supported `application/vnd.redhat.cincinnati.v1+json`, `application/vnd.redhat.cincinnati.v2+json`, etc.  So I'm not strongly against porting to using the global and dropping the `content_type` argument here.  But using both pathways at once seems like it would be too confusing.